apps/meteor/server/models/raw/Subscriptions.ts

```diff
@@ -594,6 +605,21 @@ export class SubscriptionsRaw extends BaseRaw<ISubscription> implements ISubscri
 		return this.updateMany(query, update);
 	}
 
+	updateNameAndFnameByVisitorIds(visitorIds: string[], name: string): Promise<UpdateResult | Document> {
+		const query = { 't': 'l' as const, 'v._id': { $in: visitorIds } };
+
+		const update = {
+			$set: {
+				name,
+				lowerCaseName: name.toLowerCase(),
```

Comment:
Any reason for this new property? Didn't get it...

Reply:
Hmmm I just noticed this isn't in `core-typings`, but we actually have this prop in the DB due to [this line](https://github.com/RocketChat/Rocket.Chat/blob/92b6a50c343948b48415fe81de0b1f601aa02979/apps/meteor/app/livechat/server/lib/Helper.ts#L265) and we rely on it on the client side
@KevLehman do you recall what was the idea for this? And why we didn't include it in core-typings? :thinking: 

But I believe this prop should be updated even though it isn't in `core-typings` so that we can keep consistency (and maybe add it to `core-typings` in another PR)

Reply:
Can you create a task to remove it from the `createSubscription` call as well? 🤗 so we remove it on the future

Reply:
Done, @KevLehman ! [CORE-809](https://rocketchat.atlassian.net/browse/CORE-809)